modules/cms-kit/src/Volo.CmsKit.Public.Web/CmsKitPublicWebModule.cs

```diff
@@ -61,6 +61,10 @@ public override void PostConfigureServices(ServiceConfigurationContext context)
         {
             Configure<RazorPagesOptions>(options =>
             {
+                if (GlobalFeatureManager.Instance.IsEnabled<PagesFeature>())
```

Comment:
Can we move the  if check to the top of the `Configure` method? So that we won't call the configure method unnecessarily.